addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
-const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : () => {};
+const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : (callback) => callback();
```

Comment:
Correct. If you try to do `requestAnimationFrame(null)` the browser will yell at you.